### ISSUE TITLE
🐛[BUG] Moblins using TP moves on themselves

### ISSUE THREAD

**Issue by ShadowFox85** [HorizonFFXI/HorizonXI-Issues#1454]
**What does this bug affect?**
Moblins in Oldton Movalpolos are using Paralysis Shower and Crispy Candle on themselves rather than the target who has aggro.

**Steps to Reproduce**
Fight moblins and let them use tp moves

**Expected behavior**
TP move should be used on the aggro target

**Screenshots**
![image](https://user-images.githubusercontent.com/32938187/233012578-41ad5547-d1cf-46db-8b68-b83ddbd0a502.png)

**Comment by ShadowFox85** [comment]
Just to add to this, may not actually be a bug as may be similar to goblin bomb toss where it can accidentally hurt themselves?

Prior to posting this I had seen them target themselves almost 100% of the time, but of course the second I posted this the rate dropped and it's probably closer to 50% of the time now.

**Comment by TristtHXI** [comment]
> Just to add to this, may not actually be a bug as may be similar to goblin bomb toss where it can accidentally hurt themselves?
> 
> Prior to posting this I had seen them target themselves almost 100% of the time, but of course the second I posted this the rate dropped and it's probably closer to 50% of the time now.

Moblin moves can all backfire on themselves; https://ffxiclopedia.fandom.com/wiki/Category:Moblins

**Comment by ShadowFox85** [comment]
Yeah I think it was just a case of really bad (or good depending on how you look at it) RNG. I'd been down in Oldton for a while and almost every single move them was backfiring.
